index.html

```diff
@@ -118,83 +118,91 @@ <h4 class="project-client">Facebook</h4>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
           </p>
           <ul class="pro-tool">
             <li class="tool">html</li>
-            <li class="tool">Ruby on rails</li>
             <li class="tool">css</li>
             <li class="tool">javascript</li>
           </ul>
-          <button class=" more more-about">see project</button>
+          <button class="more">see project</button>
         </div>
       </div>
-      <div class="project uber-project">
+      <div class="project multipost2-project">
         <img
-        src="image\Snapshoot Portfolio 21.png "
-        alt="tonic"
-        class="project-img"
-      />
+          src="./image/Snapshoot Portfolio4.svg "
+          alt="tonic"
+          class="project-img"
+        />
+
         <div class="project-details">
           <h2 class="project-title">
-            Uber Navigation
+            Multipost <br />
+            stories
           </h2>
           <nav class="project-nav">
-            <h4 class="project-client">Uber</h4>
+            <h4 class="project-client">Canopy</h4>
+
             <img src="./image/Counter.svg" alt="counter" />
-            <span class="project-lang">Lead Developer</span>
+            <span class="pro-lang">Back End Dev</span>
             <img src="./image/Counter.svg" alt="counter" />
-            <span class="year">2018</span>
+
+            
+
+            <span class="year">2015</span>
           </nav>
           <p class="project-description">
-            A smart assistant to make driving more safe, efficient, and fun by 
-            unlocking your most expensive computer: your car.
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
           </p>
           <ul class="pro-tool">
             <li class="tool">html</li>
-            <li class="tool">Ruby on rails</li>
             <li class="tool">css</li>
             <li class="tool">javascript</li>
           </ul>
           <button class="more">see project</button>
         </div>
-        
       </div>
     </section>
     <section class="about-myself">
       <div class="myself">
-       <div class="myself-intro"> 
-        <h2 class="name about-me">About<br/>Myself</h2>
+        <h2 class="name">About<br/>Myself</h2>
         <p class="text">
           Hello I’m a software developer! I can help you build a product ,
           feature or website Look through some of my work and experience! If you
           like what you see and have a project you need coded, don’t hestiate to
           contact me.
         </p>
 
-        
+        <div class="contact">
           <a href="#" class="connect">LET'S CONNECT</a>
           <ul class="social-icons">
+
             <li><img src="./image/Github.png" alt="github" /></li>
             <li><img src="./image/Icon - Linkedin.png" alt="ln" /></li>
 
```

Comment:
- [ ] Kindly fix the position of the social media icons, as well as the frameworks and skills section (color, position).
![review](https://user-images.githubusercontent.com/53347966/219433243-9f228e6c-387c-49dc-a00b-03cdeb762df3.JPG)